### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_ScreenReader:The name of the page on landing is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9296]
[“Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#NVDA;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader:NVDA
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch screen reader as NVDA
8. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.

**Actual:**
When the page is displayed there is no announcement of the name of the screen of the button in focus.

Issue not observed with JAWS

**Expected:**
The page on landing should announce "Service setting" and the button "next " which is in focus.

**User Impact:**
Screen reader users will not know which screen they have landed on.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/ESINHbRukShGlV-FNLrBDQ0BTQ0qU_rv16ctCOYdqk5rFA?e=7XXAeG

**Comment by alanrenmsft** [comment]
this works with voice over, might be an issue with nvda, the guideline we got for this kind of issue is as long as we are implementing the feature using the standard way, we don't need to worry about a particular screen reader. have you tried narrator?

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
